controllers/common/util.go

```diff
@@ -771,3 +772,60 @@ func FindDifference(superset, subset []string) []string {
 
 	return difference
 }
+
+// EnableMaintenanceMode enables maintenance mode for CommonService CR
+func EnableMaintenanceMode(c client.Client, masterNs string) error {
+	// Fetch CommonService CR
+	instance := &apiv3.CommonService{}
+	if err := c.Get(context.TODO(), types.NamespacedName{
+		Name:      "common-service",
+		Namespace: masterNs,
+	}, instance); err != nil && !errors.IsNotFound(err) {
+		klog.Errorf("Failed to get CommonService CR in %s: %v", masterNs, err)
+		return err
+	} else if errors.IsNotFound(err) {
+		klog.Infof("CommonService CR is not found in %s", masterNs)
+		return nil
+	} else {
+		// Set annotation: commonservices.operator.ibm.com/self-pause to true
+		if instance.ObjectMeta.Annotations == nil {
+			instance.ObjectMeta.Annotations = make(map[string]string)
+		}
+		instance.ObjectMeta.Annotations["commonservices.operator.ibm.com/self-pause"] = "true"
+	}
+
+	// Update CommonService CR
+	if err := c.Update(context.Background(), instance); err != nil {
+		klog.Errorf("Failed to update CommonService CR: %v", err)
+		return err
+	}
+	return nil
+}
+
+// DisableMaintenanceMode disables maintenance mode for CommonService CR
+func DisableMaintenanceMode(c client.Client, masterNs string) error {
+	// Fetch CommonService CR
+	instance := &apiv3.CommonService{}
+	if err := c.Get(context.TODO(), types.NamespacedName{
+		Name:      "common-service",
```

Comment:
Could we make CS CR name as a variable? So it would be passed into the function.